docs/data-sources/infoblox_ipv6_network_container.md

```diff
@@ -0,0 +1,79 @@
+# IPv6 Network Container Data Source
+
+Use the data source to retrieve the following information for an IPv6 network container resource from the corresponding
+object in NIOS:
+* `network_view`: the network view which the network container exists in. Example: `nondefault_netview`
+* `cidr`: the IPv6 network block of the network container. Example: `2002:1f93:0:2::/96`
+* `comment`: a description of the network container. This is a regular comment. Example: `Tenant 1 network container`.
+* `ext_attrs`: the set of extensible attributes of the network view, if any. The content is formatted as stirng of JSON map. Example: `"{\"Administrator\":\"jsw@telecom.ca\"}"`.
+
+As there is new feature filters, the previous usage of combination of Network view and address of the network block in CIDR format has been removed.
```

Comment:
As there are new feature filters, the previous usage of a combination of Network view and address of the network block in CIDR format has been removed.